emissionsapi/openapi.yml

```
@@ -33,6 +33,24 @@ paths:
                 $ref: '#/components/schemas/featurecollection'
         400:
           description: Parameter error
+  /api/v1/average/:
```

Maybe, to indicate that the result is JSON?

```suggestion
  /api/v1/average.json:
```

---

I probably would not add the result type to the endpoint. We could easily add other result types to the same endpoint later. Maybe the next great serializing format is already on the way :smile: 

But since we already did it in `/api/v1/geo.json`, I add this to this endpoint too. We should talk about this in detail when we think about a proper `design` for our API. Maybe in `v2`.